app/frontend/src/submissions/components/SubmissionForm/Coords.vue

```diff
@@ -295,7 +295,8 @@ export default {
       latitudeDMSValidation: false,
       coordinateLookup: new Map(),
       coordinateResolveLookup: new Map(),
-      validCoordinate: null
+      validCoordinate: null,
+      timeout: null
```

Comment:
a name like `coordCheckTimeout` or `coordCheckDebounce` or something similar might be helpful